src/test/java/posters/pageobjects/components/PosterSlider.java

```diff
@@ -0,0 +1,108 @@
+package posters.pageobjects.components;
+
+import static com.codeborne.selenide.Condition.*;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.$;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import com.xceptance.neodymium.util.Neodymium;
+import io.qameta.allure.*;
+
+
+public class PosterSlider extends AbstractComponent {
+
+	// The slides in the carousel are indexed from 1 to NUM_SLIDES
+	private static final int NUM_SLIDES = Integer.parseInt(Neodymium.localizedText("HomePage.posterSlider.numSlides"));
+	private static final String TITLE_TEXT = Neodymium.localizedText("HomePage.posterSlider.title");
+	private static final String ACTIVE_CLASS = Neodymium.localizedText("HomePage.posterSlider.activeSlideClass");
+	
+	private ElementsCollection carouselIndicators = $$("#pShopCarousel > ol.carousel-indicators > li");
+	private ElementsCollection carouselSlides = $$("#pShopCarousel > div.carousel-inner > div[id^='pShopCarouselItem']");
+	private SelenideElement prevSlideBtn = $("#btnCarouselPrev");
+	private SelenideElement nextSlideBtn = $("#btnCarouselNext");
+	private SelenideElement title = $("#titleIndex");
+	
+	@Override
+	public void isComponentAvailable() {
+		$("#pShopCarousel").should(exist);
+	}
+
+	/**
+	 * Navigate to a slide using its position
+	 * @param position the position of the slide to navigate to
+	 */
+	@Step("Open a slide using its position in the carousel")
+	public void goToSlide(int position) {
+		int positionIdx = getSlideIdx(position);
+		carouselIndicators.get(positionIdx).click();
+	}
+
+	@Step("Go to the previous slide")
+	public void goToPrevSlide() {
+		prevSlideBtn.click();
+	}
+
+	@Step("Go to the next slide")
+	public void goToNextSlide() {
+		nextSlideBtn.click();
+	}
+
+	/**
+	 * Returns a slide index in the range of 0 and NUM_SLIDES - 1 using modular arithmetic
+	 * @param position of the slide
+	 * @return
+	 */
+	private int getSlideIdx(int position) {
+		return Math.floorMod(--position, NUM_SLIDES);
+	}
+
+	/**
+	 * Get the position of the currently visible slide
+	 * @return slide's position in range 1 to NUM_SLIDES
+	 */
+	@Step("Validate exactly 1 slide is visible at the instant and get its position")
```

Comment:
the validation must not be named, since this is part of correctness of the function in this case